content/Api/Blip.Api.Template.Facades/Strategies/ExceptionHandlingStrategies/ExceptionHandlingStrategy.cs

```diff
@@ -1,11 +1,47 @@
-using System;
+﻿using System;
 using System.Threading.Tasks;
+
 using Microsoft.AspNetCore.Http;
 
+using Serilog;
+
+using Blip.Api.Template.Models;
+
 namespace Blip.Api.Template.Facades.Strategies.ExceptionHandlingStrategies
 {
     public abstract class ExceptionHandlingStrategy
     {
-        public abstract Task<HttpContext> HandleAsync(HttpContext context, Exception exception);
+        #region Private Fields
+
+        private readonly ILogger _logger;
+
+        #endregion Private Fields
+
+        #region Public Constructors
+
+        protected ExceptionHandlingStrategy(ILogger logger)
+        {
+            _logger = logger;
+        }
+
+        #endregion Public Constructors
+
+        #region Public Methods
+
+        public async Task<HttpContext> HandleAsync(HttpContext context, Exception exception)
```

Comment:
_for now_ all implementations are the same, but the user might want to implement it's own version of the method. While it is technically changeable here, it would affect everything.

If this is to be maintained, it needs the `virtual` modifier on the method, so that subclasses can `override` the implementation

Reply:
Great observation! Done!